schedule/yast/opensuse/agama_autoinst.yaml

```diff
@@ -0,0 +1,10 @@
+---
+name: agama_auto
+description: >
+  First test for autoinstallation wit agama
```

Comment:
s/wit/with/
Please don't call it first, just states what does in the description prepare blab, boot, perform auto-installation blablab.